api/v1alpha3/conversion_test.go

```diff
@@ -42,6 +43,78 @@ func AWSClusterStaticIdentityFuzzer(obj *AWSClusterStaticIdentity, c fuzz.Contin
 	obj.Spec.SecretRef.Namespace = ""
 }
 
+func fuzzFuncForAWSCluster(_ runtimeserializer.CodecFactory) []interface{} {
+	return []interface{}{
+		AWSClusterFuzzer,
+	}
+}
+
+func AWSClusterFuzzer(obj *v1alpha4.AWSCluster, c fuzz.Continue) {
+	c.FuzzNoCustom(obj)
+
+	if obj.Status.Bastion == nil {
+		obj.Status.Bastion = &v1alpha4.Instance{}
+	}
+	obj.Status.Bastion.RootVolume = &v1alpha4.Volume{Encrypted: aws.Bool(false)}
```

Comment:
```suggestion
	obj.Status.Bastion.RootVolume = &v1alpha4.Volume{Encrypted: pointer.BoolPtr(false)}
```

Only use aws.Bool when working with AWS services.